apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -1347,11 +1346,8 @@ func (m *ModelManagerAPI) ModifyModelAccess(ctx context.Context, args params.Mod
 				},
 				Access: modelAccess,
 			},
-			AddUser:  true,
-			External: &external,
-			ApiUser:  user.NameFromTag(m.apiUser),
```

Comment:
issue: the facade now needs to check that the apiUser has permission sufficient to change the permissions.